### DISCUSSION TITLE
ImageUpdate fails with unknown error and no additional info

### DISCUSSION THREAD

**Discussion by jonaskello** [fluxcd/flux2#876] [Q&A] [answered]
We have a self-hosted GitLab CE instance and I'm trying to get `ImageUpdateAutomation` to work. Both `ImageRepository` and `ImagePolicy` works and I can see they resolve to the correct image. However the last part with automatic update fails without much info to go on as seen below.

Perhaps it is somehow related to authentication for writing. The deploy key to the `GitRepository` used has write access as far as I can see.  It is not the `GitRepository` where flux is installed but a separate tenant repo.

Is there somewhere I can get more error information to troubleshoot this?

```
❯ k describe imageupdateautomations.image.toolkit.fluxcd.io
...
Events:
  Type    Reason  Age                   From                         Message
  ----    ------  ----                  ----                         -------
  Normal  error   21m (x26 over 127m)   image-automation-controller  unknown error: remote:
  Normal  error   3m55s (x18 over 15m)  image-automation-controller  unknown error: remote:
```
```
❯ k -n flux-system logs image-automation-controller-68fd65c6fc-wrtp9
...
runtime.manager.controller.imageupdateautomation","msg":"Reconciler error","reconciler group":"image.toolkit.fluxcd.io","reconciler kind":"ImageUpdateAutomation","name":"flux-nibe","namespace":"XXX","error":"unknown error: remote: "}
```

**Comment by stefanprodan** [comment, marked as answer]
Duplicate of https://github.com/fluxcd/image-automation-controller/issues/102
